Detectors/TRD/workflow/src/trd-tracking-workflow.cxx

```diff
@@ -53,6 +53,7 @@ void customize(std::vector<o2::framework::ConfigParamSpec>& workflowOptions)
     {"enable-qc", VariantType::Bool, false, {"enable tracking QC"}},
     {"enable-pid", VariantType::Bool, false, {"Enable PID"}},
     {"enable-ph", VariantType::Bool, false, {"Enable creation of PH plots"}},
+    {"trd-digits-spec", VariantType::Int, 0, {"Input digits subspec, ignored if disable-root-input is false"}},
```

Comment:
This change should be obsolete after https://github.com/AliceO2Group/AliceO2/pull/12181is merged

Reply:
I was mistaken so I think this is indeed needed